integration-cli/docker_cli_rm_test.go

```diff
@@ -63,13 +65,9 @@ func (s *DockerSuite) TestRmContainerOrphaning(c *check.C) {
 	if _, err := buildImage(img, dockerfile2, true); err != nil {
 		c.Fatalf("Could not rebuild image %s: %v", img, err)
 	}
-	// try to remove the image, should error out.
```

Comment:
You removed this because now it will successfully untags and remove the child without error while leaving the parent intact? Why did this error out before